tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -103,6 +103,8 @@ cc: cc-kernel-tarball \
 	cc-rootfs-image-tarball \
 	cc-virtiofsd-tarball \
 	cc-shim-v2-tarball \
+	cc-sev-kernel-tarball \
+	cc-snp-qemu-tarball \
```

Comment:
These do not belong here. `cc-sev-kernel-tarball` is not needed in this list at all (https://github.com/kata-containers/kata-containers/pull/6416/commits/72672376600ff49e5604f28b54737c6985bb1e03) and `cc-snp-qemu-tarball` should be in the `EXTRA_TARBALL` list at the top of this file.